### ISSUE TITLE
Unusable source maps written to `dist/`

### ISSUE THREAD

**Issue by AprilArcus** [isaacs/minimatch#203]
The source map URLs written to dist take the form:

```
//# sourceMappingURL=brace-expressions.js.map
```

and the corresponding file contains a reference to the corresponding source file `src/**/*.ts`.
```
{
  "version": 3,
  "file":"brace-expressions.js",
  "sourceRoot":"",
  "sources": ["../../src/brace-expressions.ts"],
  ...
}
```

since this is not included in the package as published to `npm`, the source map cannot be consumed by e.g. webpack's [source-map-loader](https://webpack.js.org/loaders/source-map-loader/), and if not so consumed or stripped out, produce console errors in development mode.

My team encountered a similar issue in `ag-grid` last year, for rererence: https://github.com/ag-grid/ag-grid/issues/5039

**Comment by koshic** [comment]
@AprilArcus you mean https://www.typescriptlang.org/tsconfig#inlineSources missing in tsconfig-base.json?

**Comment by isaacs** [comment]
This should be fixed in fc3fa85, published on 7.4.3
